docs/react/guides/window-focus-refetching.md

```diff
@@ -105,7 +88,7 @@ useEffect(() => {
 
 ## Managing focus state
 
-[//]: # 'Example5'
+[//]: # 'Example4'
```

Comment:
maybe we should switch to named examples @DamianOsipiuk 😅. I see the vue-query docs only overwrite/use the `'Example'` slot, but this could become tedious with numbers

Reply:
As you know naming things is hard, so i just went with numbers 🙈 
But yeah, probably we should name them properly.